docs/internal-developers/rest-api/README.md

```diff
@@ -0,0 +1,5 @@
+# REST API
+
+| Document                                                                         | Description |
+|----------------------------------------------------------------------------------| ---------------------------------------------------------------------------------- |
```

Comment:
Nitpick: Based on https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables, I was under the impressions that Markdown tables have this format:
```md
| First Header  | Second Header |
| ------------- | ------------- |
| Content Cell  | Content Cell  |
| Content Cell  | Content Cell  |
```
Note the space between `|` and `---`. I'm using the [Markdown All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) extension for VS Code to automatically format tables. Saving this file on my end, leads to the following output:

```md
# REST API

| Document                                                                | Description                                                                        |
| ----------------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
| [Adding an endpoint to ExtendSchema](./extend-rest-api-new-endpoint.md) | A step by step process for contributors to expose a new endpoint via ExtendSchema. |
```

Based on https://www.jetbrains.com/help/phpstorm/code-style-markdown.html#wrapping-and-braces, it appears that PHPStorm should also be capable of formatting Markdown tables automatically. That said, this is a nitpick, so feel free to leave this file as it is. 😉

Reply:
Hmm, I don't think there's an option to have the space between the pipe and the word on PHPStorm, I looked but couldn't find it. I will copy what you entered and use that instead. I have to disable automatic table formatting to prevent it from reverting 😁